server/postgresql/seed/photos_seed.js

```diff
@@ -0,0 +1,22 @@
+const faker = require('faker');
+const fs = require('fs');
+
+
+    var titles = "id,dish_id,caption,source,restaurant_id \n";
+    var data = "";
+    for (var i = 0; i < 100000; i++) {
+        var id = i.toString() + ',';
+        var dish = Math.round(Math.random() * (100000 - 1) + 1).toString() + ',';
+        var caption = faker.lorem.sentence() + ',';
+        var source = `https://loremflickr.com/320/240?lock=${Math.round(Math.random() * (1000 - 1) + 1)}` + ',';
```

Comment:
are you going for a weighted random distribution? if not replace Math.round with Math.floor